.circleci/config.yml

```diff
@@ -154,6 +156,20 @@ jobs:
       - run: yarn install --cwd=dashboard --frozen-lockfile
       - run: yarn --cwd=dashboard run lint
       - run: yarn --cwd=dashboard run test --maxWorkers=4 --coverage
+  test_chart:
+    docker:
+      - image: circleci/golang:1.9
+    environment:
+      HELM_VERSION: v2.9.1
+    steps:
+      - checkout
+      - run: |
```

Comment:
I would move the installation of helm to a "definition" (without the `helm init` to avoid duplicity)

Reply:
:+1: yep was thinking about doing that